libs/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('DataLayerColorService', () => {
       };
       service.chooseColorsFromPalette(layer);
       expect(layer.datasets[0].borderColor).toEqual(palette[0]);
-      expect(layer.datasets[1].borderColor).toEqual(palette[0]);
+      expect(layer.datasets[1].borderColor).toEqual(palette[1]);
```

Comment:
This test description is `it('should reuse color from matching dataset'`. This change makes it expect the opposite.
Please update the test so it correctly tests the new dataset matching code.